src/core/geometry/qgslinestring.h

```diff
@@ -58,6 +59,204 @@ class CORE_EXPORT QgsLineString: public QgsCurve
      */
     QgsLineString( const QVector<QgsPoint> &points ) SIP_HOLDGIL;
```

Comment:
this method is not exposed to sipify no?
```suggestion
    QgsLineString( const QVector<QgsPoint> &points );
```